scripts/gen.py

```diff
@@ -80,10 +80,16 @@ class ArgTemplate(string.Template):
 _XPARSER = lark.Lark(
     _GRAMMAR, parser='lalr', propagate_positions=True, keep_all_tokens=True)
 
-# _FN_WHITELIST & _FN_BLACKLIST takes either name or mapsig.
+# _FN_WHITELIST/_FN_FULL_OVERRIDE/_FN_BLACKLIST takes either name or mapsig.
 _FN_BLACKLIST = set([])
 
-_FN_WHITELIST = set([
+# List of non-leaf ops we want to override both forward + backward.
+_FN_FULL_OVERRIDE = set([
+    'max_pool2d(Tensor, IntArrayRef, IntArrayRef, IntArrayRef, IntArrayRef, bool) -> Tensor',
+])
+
+# List of non-leaf ops we want to override forward.
+_FN_WHITELIST = _FN_FULL_OVERRIDE | set([
```

Comment:
Why can't those (copy_ & Co.) be marked as leaf by the pytorch codegen?

Reply:
It will require changes to other backends as well. I have plan to change some of them but it's lower pri in my queue for now.